hermes_eea/calibration/calibration.py

```diff
@@ -1,17 +1,27 @@
 """
 A module for all things calibration.
 """
+from datetime import datetime, timezone, timedelta
 import random
 import os.path
 from pathlib import Path
-
+import sys
 import ccsdspy
+import numpy as np
+from spacepy import pycdf
 
 from hermes_core import log
 from hermes_core.util.util import create_science_filename, parse_science_filename
-
 import hermes_eea
 from hermes_eea.io import read_file
+import hermes_eea.calibration as calib
+from hermes_eea.io.EEA import EEA
```

Comment:
Modules should have short, all-lowercase names. Also given that this is the hermes_eea package, all of the contents of this package are eea relevant so an eea module is confusing. Not clear what it might provide. It could be anything!

Reply:
In our lab's convention. this returns a model of the data before we put it in the cdf. We capitalize it...